Products/Zuul/facades/devicefacade.py

```diff
@@ -581,8 +581,13 @@ def _moveDevices(self, uids, target):
                 dev.setLocation(targetname)
                 notify(ObjectAddedToOrganizerEvent(dev, target))
         elif isinstance(target, DeviceClass):
-            exports = self._dmd.Devices.moveDevices(targetname,[dev.id for dev in devs])
-        return exports
+            moved_devices_count = self._dmd.Devices.moveDevices(targetname,[dev.id for dev in devs])
+
+        result = {
+            'success': bool(moved_devices_count),
+            'message': 'The %s devices have been moved' % moved_devices_count
+        }
```

Comment:
What's the reason for the changing the result from a number to a `dict` object?  As far as I can tell, callers to this function either ignore the result or only care that `bool(result) is True`.